explainers/subresource-loading.md

```diff
@@ -154,6 +155,35 @@ the bundle, and a subframe is instantiated as an
 
 Note that `resources` attribute is reflected to JavaScript as a [`DOMTokenList`](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList).
 
+## Request's mode and credentials mode
+
+With `<link>`-based API, a
+[request](https://fetch.spec.whatwg.org/#concept-request) for a bundle
+will have their [mode][request mode] set to "`no-cors`" and their
+[credentials mode][credentials mode] set to "`include`" unless a
+[crossorigin][crossorigin attribute] attribute is specified.
+
+If a [crossorigin][crossorigin attribute] attribute is specified,
+their [mode][request mode] and [credentials mode][credentials mode]
+will be set as the [CORS settings attributes][cors settings attribute]
+section describes.
+
+The following table is the summary.
+
+| State          | [mode][request mode] | [credentials mode][credentials mode] |
+| --------------- | -----------          | ----------------                     |
+| No CORS         | "`no-cors`"          | "`include`" [1]                       |
```

Comment:
Can we follow the ["For more modern features"](https://html.spec.whatwg.org/multipage/urls-and-fetching.html#cors-settings-attributes:~:text=For%20more%20modern%20features) policy, and always use CORS for bundle fetching?

Reply:
As per the separate discussion, we've decided to follow "For more modern features" policy.

Now we use "cors" always, and use "same-origin" for a credentials mode if `crossorigin`attribute is not given.

Reply:
Thanks! And sorry for forgetting why we had `no-cors` in the first place.